### ISSUE TITLE
[Feature Request] Simple CLI tool to validate package names

### ISSUE THREAD

**Issue by aarondill** [npm/validate-npm-package-name#65]

**Comment by CodeVibranium** [comment]
Hey @aarondill , is this issue still open?

**Comment by aarondill** [comment]
yes. although it seems like maybe this repo is in maintenance-only mode?

**Comment by CodeVibranium** [comment]
> yes. although it seems like maybe this repo is in maintenance-only mode?
@aarondill , If there are any issues to fix please let me know.

**Comment by broofa** [comment]
@lukekarrys: Would you be open to a PR for this?  Any opinions about what the name of the CLI should be?

**Comment by CodeVibranium** [comment]
Sure, I'm
CLI name, give me some time @broofa

**Comment by lukekarrys** [comment]
I think the current package name, although a little verbose, makes the most sense.

A good pattern to follow would be this repo where I recently separated the index and bin script https://github.com/npm/installed-package-contents/pull/72
